src/tests/ftest/io/mdtest_small.yaml

```diff
@@ -7,7 +7,7 @@ hosts:
   test_clients:
     - client-E
     - client-F
-timeout: 800
+timeout: 1500
```

Comment:
Why increasing timeout values? Is something going which warrants big increase (eg: scm_size??)?

Reply:
Same as above. Not changing it yet.